### DISCUSSION TITLE
Works in chrome, but not firefox

### DISCUSSION THREAD

**Discussion by pyromanci** [vladmandic/human#466] [General]
I was looking at your faceid demo https://vladmandic.github.io/human/demo/faceid/. When using the demo in firefox it throws an 
`Uncaught (in promise) Error: Kernel 'Draw' not registered for backend 'webgl'` When i run it chrome it works. It's the `await human.tf.browser.draw(current.face.tensor, dom.canvas);` command located in the detectFace() function that does it. 

Checking hardware acceleration is enabled. webgl is enabled. Firefox version 126.0 if that matter. I've tired it on multiple computers and fresh installs. As I saw in another post. Firefox is suppose to be supported. So i'm not sure what the problem could be.

**Comment by vladmandic** [comment]
firefox is supported by library itself, doesn't mean there is time/resources to test every demo option.

there are other ways to draw tensor to canvas, that particular demo uses `human.tf.browser.draw(current.face.tensor, dom.canvas);` which is causing problems for firefox. at the end, that is a demo draw function, not a core library item.
